packages/01-host/public/index.html

```diff
@@ -1,10 +1,24 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <script src="http://localhost:3002/remoteEntry.js"></script>
-    <script src="http://localhost:3003/remoteEntry.js"></script>
+    <script type="systemjs-importmap">
+      {
+        "imports": {
+          "app_one": "http://localhost:3001/remoteEntry.js",
+          "app_two": "http://localhost:3002/remoteEntry.js",
+          "app_three": "http://localhost:3003/remoteEntry.js",
+          "app_one/": "http://localhost:3001/",
+          "app_two/": "http://localhost:3002/",
+          "app_three/": "http://localhost:3003/"
+        }
+      }
+    </script>
   </head>
   <body>
     <div id="root"></div>
+    <script src="https://unpkg.com/systemjs/dist/system.js"></script>
+    <script>
```

Comment:
Don't you also need to call `System.import('app_one')`? How is the remoteEntry file being loaded?